.github/workflows/ci.yml

```diff
@@ -54,3 +54,32 @@ jobs:
     # - name: Exit if coverage condition not met
     #   if: ${{ steps.test.outputs.exit_code }} != 0
     #   run: exit ${{ steps.test.outputs.exit_code }}
+    
+    - name: Cypress install
+      uses: cypress-io/github-action@v6
+      with:
+        runTests: false
+        build: npm run build
```

Comment:
Shouldn't this step just be `npm run build` by itself? Is the Cypress action required here?